### DISCUSSION TITLE
Panel statistic information or Total Count for Table View

### DISCUSSION THREAD

**Discussion by fedoranvar** [grafana/grafana#38304] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
In general, to be able show some panel statistic information on dashboard.

To be specific, i want to display total count of rows in table panel. 
something like
![image](https://user-images.githubusercontent.com/52028706/122372393-10d0a100-cf7a-11eb-8fe7-cea931b50e62.png)

p.s. found some workaround: write variables in panel name, so for that i need a variable with value of total records in table
![image](https://user-images.githubusercontent.com/52028706/122373831-3d38ed00-cf7b-11eb-9821-47d71be7948e.png)

**Why is this needed**:
To know some info statistic info about panel (total count rows)

**Comment by jackw** [comment]
@fedoranvar thanks for opening this issue. Just to confirm, you're asking for some of the information that can be found in Edit Panel -> Query Inspector -> Stats

<img width="524" alt="Screenshot 2021-07-06 at 12 13 56" src="https://user-images.githubusercontent.com/73201/124583843-d27d2200-de53-11eb-9172-2f7f5155431c.png">

to be visible for each panel when viewing the dashboard?

**Comment by fedoranvar** [comment]
in this case - yes, number of rows.

But ideally i see some area on panel, like 'legends' of graphics, where info stats displayed, and maybe there can be done some transformations on it

for example we can show total number rows, but also we can apply some functions on it.

Maybe this stats can be accessed by variables and also there is some operations with variables, 
for example: 
 ${ $totalRequestTime - $dataProcessingTime } - substracting data processing time from total request time
 
 Thank you!

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Reply by DucatiDan** [reply]
This is actually fairly easy to do with a query e.g.  SELECT COUNT(*) FROM xxxxx WHERE condition.  However a more useful value to be able to display for a table is the rows currently visible. Now that we have the filter function for columns it would be great to see the number of rows after filtering. Manually counting the rows to see the results of the filter is not enjoyable

**Reply by mtettke123** [reply]
looking for exactly this. A number of the rows visible when using filters, etc. Any news on this?

**Reply by avinashkris9** [reply]
> looking for exactly this. A number of the rows visible when using filters, etc. Any news on this?

Any hope or way to do this  on new releases ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
